src/wp-includes/interactivity-api/interactivity-api.php

```diff
@@ -66,7 +66,11 @@ function wp_interactivity_process_directives_of_interactive_blocks( array $parse
 
 	return $parsed_block;
 }
-add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks' );
+/*
+ * Uses a priority of 20 to ensure that other filters can add additional attributes to
+ * $parsed_block before the processing starts.
+ */
+add_filter( 'render_block_data', 'wp_interactivity_process_directives_of_interactive_blocks', 20, 1 );
```

Comment:
The value of 20 is a bit arbitrary. What if someone changes block attributes at priority 30?

To be fair, this boils down to a common problem, which is: How can I run my hook _after_ all other hooks? There's no good answer to that I think. But in principle, using `PHP_INT_MAX` is the most reliable choice. I don't think there are any downsides to using that, but if for some reason there's a preference not to go with that, then maybe at least something like `100`, `1000`, or `9999`, those are values I've seen quite a bit used for such a use-case.

This isn't worth having a big discussion about as part of this PR, but I just feel 20 is far too low for something that should run after other filter callbacks.

Reply:
I've seen PHP_INT_MAX in other filters, so I guess is fine here too.